SplitFiles/bread.c

```diff
@@ -55,9 +55,9 @@ BreadState* get_bread(BreadBin* bin, unsigned char typeLetter) {
     }
     BreadState* newBread = malloc(sizeof(BreadState));
     newBread->temperature = 0;
-    newBread->moisture = type->moisture + (rand() % type->moistureVariability);
+    newBread->moisture = type->moisture + (rand() & type->moistureVariability);
```

Comment:
The big drag on bread creation is the sound - saving a couple of cycles here at the expense of imposing a hidden requirement on the code (variability must be a power of 2 - 1) doesn't feel like a great trade-off.  This late in the day it's fine I suppose, but this feels like change for its own sake.